app/src/main/res/values/strings.xml

```diff
@@ -138,5 +138,6 @@
     <string name="add_an_event">Add an event</string>
     <string name="cycle_view">Cycle view</string>
     <string name="tl_screen_new_task">New task</string>
+    <string name="show_qr_code">show QR code</string>
```

Comment:
As all other strings have it I would put an uppercase at the start of this new one but this is nitpicking

Reply:
it might be nitpicky but its good to keep conventions, ill update it in the next commits